### PR TITLE
Fix: Tactical Nuke Mig Upgrade Not Buildable While Low On Power

### PR DESCRIPTION
**Pull request by commy2**
ZH

- The Tactical Nuke Mig upgrade cannot be purchased from the Nuclear Missile Silo while low on power. Neutron Shells, as well as Nuclear Tanks and Uranium Shells for other factions, can still be bought while low on power.
- If the upgrade has already been queued, but is not finished yet, it can still progress to completion even when low on power.
- When the building is disabled by an EMP strike however, none of the upgrades can be purchased and their progress will freeze... Except when the building is low on power AND disabled by an EMP strike: Then none of the upgrades can be purchased, but already queued upgrades will progress... The progress will freeze again once the building has enough power, but is still disabled by EMP... ...

After patch:

- The Tactical Nuke Mig upgrade behaves like every other upgrade on the Nuclear Missile Silo.

Note:

- Apparently, in an earlier build of the game, Tactical Nuke Mig was a Propaganda Center upgrade, while Isotope Stability was a Nuke Silo upgrade. I moved the flag away from the Isotope Stability, because a Propaganda Center, while it consumes power, cannot be low on power (not to be confused with "disabled" by EMP).

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/CommandButton.ini [hunk 1]
```diff
@@ -6521,7 +6522,6 @@ End
 CommandButton Nuke_Command_UpgradeChinaIsotopeStability
   Command       = PLAYER_UPGRADE
   Upgrade       = Upgrade_ChinaIsotopeStability
-  Options       = IGNORES_UNDERPOWERED
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/CommandButton.ini:6524]: Why is this removed?

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/CommandButton.ini:6524]: This upgrade appears on the Propaganda Center. The Propaganda Center cannot be low on power: if the player is low on power, the Propaganda Center continues operation, so this flag does nothing.

The Propaganda Center can only be disabled by EMP strikes or Microwave, but that is not covered by this flag.

It does nothing, so I removed it.

**Review by xezon — APPROVED** [review]
